household_account_book_app/app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  authenticates_with_sorcery!
+  validates :password, length: { minimum: 5 }
+  validates :password, confirmation: true
```

Comment:
細かいですが、同じ属性への validates は以下のようにまとめられるかと思います！
```suggestion
  validates :password, length: { minimum: 5 }, confirmation: true
```